Standards/CCSMB-5.md

```diff
@@ -0,0 +1,23 @@
+# CCSMB-5: Radio Transmissions
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+The server will communicate to the client over modem
+
+## Packet
+A packet consists of the following fields in a table:
+- buffer: A table that contains Signed 8-bit PCM @ 48kHz audio
+- station: A string that contains the name of the radio station
```

Comment:
Instead of using the station name to identify the source, the computer's ID should be used, as this is unique per source (assuming servers are being truthful).

Reply:
That's a better solution than the earlier proposed random IDs.

Reply:
I'll keep the station in there for metadata as that seems appropriate, but the id thing is pretty good